cmake/modules/hunter_add_repository.cmake

```diff
@@ -0,0 +1,81 @@
+include(CMakeParseArguments) # cmake_parse_arguments
+include(hunter_internal_error)
+include(hunter_fatal_error)
+include(hunter_test_string_not_empty)
+include(hunter_status_debug)
+include(hunter_status_print)
+include(hunter_install_repository)
+
+function(hunter_add_repository)
+  if(PROJECT_NAME OR NOT HUNTER_GATE_ROOT)
+    hunter_fatal_error(
```

Comment:
`WIKI` missing here, see https://github.com/ruslo/hunter/wiki/dev.modules#hunter_fatal_error. You can improve this later and use `hunter_internal_error` for now.